### PR TITLE
Am national parks

### PR DESCRIPTION
**Pull request by amccadams**
Add CTA to alert and make ebook set HTML ready.

### CODE REVIEW

File: sass/settings/_colors.scss [hunk 1]
```diff
@@ -23,6 +23,8 @@ $color-light-slate: #e4eaf8 !default;
 $active-background-color: #f4f5f8 !default;
 $color-light-blue: #bdcee3 !default;
 $color-icon-blue: #2095cf !default;
+$color-lightblue: #e9f2f8 !default;
```
Comment (nicrocs) [inline sass/settings/_colors.scss]: sinc there already is a light blue, maybe we should name this something else? pale-blue?

File: src/components/modal/ebook/ebook.hbs [hunk 2]
```diff
@@ -11,7 +11,7 @@
     <h1>{{ebook.strapline}}</h1>
   </div>
   <div class="ebook__copy">
-    {{ebook.copy}}
+    {{{ebook.copy}}}
```
Comment (nicrocs) [inline src/components/modal/ebook/ebook.hbs]: should we move this to marketing and have best in europe point there instead?

Comment (amccadams) [inline src/components/modal/ebook/ebook.hbs]: I think so.  Can we wait until after we get National Parks taken care of?

Comment (nicrocs) [inline src/components/modal/ebook/ebook.hbs]: yeah, works for me

**Comment by nicrocs** [comment]
👍

**Pull request merged**